examples/l1-resources/stream-connection.ts

```diff
@@ -0,0 +1,62 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { CfnStreamConnection, CfnStreamConnectionPropsType } from 'awscdk-resources-mongodbatlas'
+import { env } from 'node:process';
+
+interface AtlasStackProps {
+	readonly projectId: string;
+	readonly profile: string;
+	readonly instanceName: string;
+	readonly connectionName: string;
+	readonly type: CfnStreamConnectionPropsType;
+	readonly clusterName: string;
+}
+
+const app = new cdk.App();
+
+
+export class CdkTestStack extends cdk.Stack {
+	constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+		super(scope, id, props);
+
+		const atlasProps = this.getContextProps();
+		const streamConnection = new CfnStreamConnection(this, "stream-connection-testing-stack", {
+			profile: atlasProps.profile,
+			instanceName: atlasProps.instanceName,
+			projectId: atlasProps.projectId,
+			connectionName: atlasProps.connectionName,
+			type: atlasProps.type,
+			clusterName: atlasProps.clusterName
```

Comment:
since it is using clusterName I would leave the type fixed to value "Cluster" as this would be a cluster stream connection example

Reply:
Done!